addons/point_of_sale/models/pos_payment_method.py

```diff
@@ -0,0 +1,92 @@
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+
+
+class PosPaymentMethod(models.Model):
+    """ Used to classify pos.payment.
+
+    Generic characteristics of a pos.payment is described in this model.
+    E.g. A cash payment can be described by a pos.payment.method with
+    fields: is_cash_count = True and a cash_journal_id set to an
+    `account.journal` (type='cash') record.
+
+    When a pos.payment.method is cash, cash_journal_id is required as
+    it will be the journal where the account.bank.statement.line records
+    will be created.
+    """
+
+    _name = "pos.payment.method"
+    _description = "Point of Sale Payment Methods"
+    _order = "id desc"
+
+    name = fields.Char(string="Payment Method", required=True)
+    receivable_account_id = fields.Many2one('account.account',
```

Comment:
remove the default, and make it 'company_dependent=True' that's how it will be implemented.
also name it 'property_receivable_account_id'